app/controllers/api/base_controller/action.rb

```diff
@@ -3,12 +3,12 @@ class BaseController
     module Action
       private
 
-      def api_action(type, id, options = {})
+      def api_action(type, id)
         klass = collection_class(type)
 
-        result = yield(klass) if block_given?
+        result = yield(klass)
 
-        add_href_to_result(result, type, id) unless options[:skip_href]
```

Comment:
`options[:skip_href]` was never used.

But we sometimes put in the parent href into the response. So putting in this caveat allows us to use api_action in more instances (in the parent cases, we were basically rewriting this function)